lib/buildsystems/autotools.rb

```diff
@@ -38,4 +38,18 @@ def self.check
     puts 'Testing with make check.'.orange if @run_tests
     system 'make', 'check' if @run_tests
   end
+
+  def self.postinstall
+    if @gnome
+      puts 'Running Gnome post_installs...'.orange
+      system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas" if File.file?("#{CREW_PREFIX}/bin/glib-compile-schemas")
+      system "gio-querymodules #{CREW_LIB_PREFIX}/gio/modules" if File.file?("#{CREW_PREFIX}/bin/gio-querymodules")
+      system "gtk-update-icon-cache -ft #{CREW_PREFIX}/share/icons/*", exception: false if File.file?("#{CREW_PREFIX}/bin/gtk-update-icon-cache")
+      system "update-mime-database #{CREW_PREFIX}/share/mime" if File.file?("#{CREW_PREFIX}/bin/update-mime-database")
+      return unless File.file?("#{CREW_PREFIX}/bin/gdk-pixbuf-query-loaders")
+
+      system 'gdk-pixbuf-query-loaders',
+             '--update-cache'
```

Comment:
Why not add this to `lib/package.rb` postinstall too?

Reply:
Or better yet, add it to lib as a helper function and require to call one-liners in these buildsystems libraries?

Reply:
I'm wondering if this should be pulled out and dropped into its own thing, like `print_source_bashrc`...

Reply:
That's what I was thinking.